core/block_drag_surface.js

```diff
@@ -229,7 +229,7 @@ Blockly.BlockDragSurfaceSvg.prototype.getCurrentBlock = function() {
  * @return {!Blockly.utils.Coordinate} The amount the workspace has been moved.
```

Comment:
I would add a comment that you're returning a copy on purpose so someone doesn't try to come "fix" it like I did haha

Reply:
Done.